dev/com.ibm.ws.jaxrs.2.0_fat/bnd.bnd

```diff
@@ -91,6 +91,7 @@ tested.features: \
   com.ibm.websphere.javaee.jsonb.1.0,\
   com.ibm.websphere.javaee.servlet.3.1,\
   com.ibm.websphere.javaee.validation.1.1,\
+  com.ibm.ws.logging.core,\
```

Comment:
I don't think you actually added any new logging to this project, so I don't think this line is necessary.

Reply:
agreed.